app-server/app/src/main/scala/user/User.scala

```diff
@@ -2,40 +2,65 @@ package com.azavea.rf.user
 
 import scala.concurrent.{Future, ExecutionContext}
 import scala.util.{Try, Success, Failure}
+import scala.language.postfixOps
 import java.sql.Timestamp
-import slick.lifted.{ColumnOrdered, TableQuery, Query}
+import slick.lifted.Query
 import com.lonelyplanet.akka.http.extensions.{PageRequest, Order}
-import de.choffmeister.auth.common.JsonWebToken
 
-import com.azavea.rf.datamodel.latest.schema.tables.{Users, Organizations}
-import com.azavea.rf.datamodel.latest.schema.tables.UsersRow
+import com.azavea.rf.datamodel.latest.schema.tables.{Users, Organizations, UsersToOrganizations}
+import com.azavea.rf.datamodel.latest.schema.tables.{UsersRow, UsersToOrganizationsRow}
 import com.azavea.rf.utils.{Database,PaginatedResponse}
 
+object Roles extends Enumeration {
+  val Uploader = Role("uploader")
+  val Viewer = Role("viewer")
+  val Admin = Role("admin")
+
+  def Role(name: String): Value with Matching =
+    new Val(nextId, name) with Matching
+
+  def unapply(s: String): Option[Value] =
+    values.find(s == _.toString)
+
+  trait Matching {
+    def unapply(s: String): Boolean =
+      (s == toString)
+  }
+}
 
 /**
-  * Instantiated from a POST request to create a user
+  * Instantiated from a POST request to create a user. Typically used with the default public org.
   */
-case class UsersRowCreate(authId: String, organizationId: java.util.UUID) {
-  def toUsersRow(): UsersRow = {
+case class UsersRowCreate(id: String, organizationId: java.util.UUID, role: Option[String] = Some("uploader")) {
+  def toUsersOrgTuple(): (UsersRow, UsersToOrganizationsRow)= {
     val now = new Timestamp((new java.util.Date()).getTime())
     val newUUID = java.util.UUID.randomUUID
-    UsersRow(
-      id=newUUID,
-      authId=authId,
-      organizationId=organizationId
+    val user = UsersRow(
+      id=id
     )
+    val defaultedRole = role getOrElse { "null" } match {
+      case Roles(role) => role
+      case s: String => throw new Exception( "\"" + s + "\" is not a valid user Role")
+    }
+
+    val userToOrg = UsersToOrganizationsRow(
+      userId=id,
+      organizationId=organizationId,
+      defaultedRole.toString(),
+      now,
+      now
+    )
+    (user, userToOrg)
   }
 }
 
+case class OrganizationWithRole(id: java.util.UUID, name: String, role: String)
 
```

Comment:
I think we've been settling on two lines between top-level objects/classes and one line inside classes/objects separating functions